JavaScript/Hashnode_Scripts/script.js

```diff
@@ -0,0 +1,71 @@
+// all requires
+const fetch = require("node-fetch");
+const ObjectsToCsv=require("objects-to-csv");
+
+
+//variables required
+var page=0;
```

Comment:
`let` and `const` is a good JS practice. Replace all `var` with `let` for better code readability and flexibility.